apps/examples/testcase/tc_main.c

```diff
@@ -71,6 +71,10 @@
 #endif
 #endif
 
+#ifdef CONFIG_LIBCXX_UTC
```

Comment:
You should put this inside conditional.
```
#ifdef
#else
#ifdef CONFIG_LIBCXX_UTC
..
#endif
#endif
```

Reply:
Under #ifdef CONFIG_TASH ? . Ok I will add it